governance.md

```diff
@@ -0,0 +1,136 @@
+
+If an existing Institutional Partner no longer has a contributing employee, they will be given a 1 year grace period for remaining employees to begin contributing.
+
+An Institutional Partner is free to pursue funding for their work on The Project through any legal means. This could involve a non-profit organization raising money from private foundations and donors or a for-profit company building proprietary products and services that leverage Project Software and Services. Funding acquired by Institutional Partners to work on The Project is called Institutional Funding. However, no funding obtained by an Institutional Partner can override The Project BDFL and Steering Council. If a Partner has funding to do Jupyter/IPython work and the Council decides to not pursue that work as a project, the Partner is free to pursue it on their own. However in this situation, that part of the Partner’s work will not be under the Jupyter/IPython umbrella and cannot use the Project trademarks in a way that suggests a formal relationship.
+
+To acknowledge institutional contributions there are two level of Institutional Partners, with associated benefits:
+
+**Tier 1** = an institution with at least one Institutional Council Member
+
+-   Acknowledged on the Jupyter/IPython websites, in talks and T-shirts.
+-   Ability to acknowledge their own funding sources on the Jupyter/IPython websites, in talks and T-shirts.
+-   Unlimited participation in the annual Institutional Partners Workshop, held during the (planned) annual Jupyter Project Retreat. This allows the Institutional Partner to invite as many of their own employees and funding sources and collaborators as they want, even if they are not project Contributors or Council Members.
+-   Ability to influence the project through the participation of their Council Member.
+-   Council Members invited to bi-annual Jupyter/IPython Developer Meeting.
+
+Existing **Tier 1** level Institutional Partners:
+
+-   UC Berkeley (Fernando Pérez, Min Ragan-Kelley, Thomas Kluyver)
+-   Cal Poly (Brian Granger)
+-   Rackspace (Kyle Kelley)
+-   Continuum Analytics (Damian Avila)
+
+**Tier 2** = an institution with at least one Institutional Contributor
+
+-   Same benefits as Tier 1 level Partners, but:
+-   Only Institutional Contributors are invited to the Institutional Partners Workshop and bi-annual Jupyter/IPython Developer Meeting.
+
+Existing **Tier 2** Partners:
+
+-   Bloomberg (Jason Grout, Sylvain Corlay)
+-   Google (Kester Tong, Kayur Patel, Mark Sandler)
```

Comment:
yep, probably... but I guess @ellisonbg wants this one merged as is and then start making the corrections... I am ok with that if that's the case...